### ISSUE TITLE
Functional furniture

### ISSUE THREAD

**Issue by bell07** [Gerold55/ma_pops_furniture#8]
This issue should collect which furniture should be functional

- [ ] Counter / Cabinets / Nightstand - like a chest
- [ ] Coffe maker - should make coffee
- [ ] Computer - Should be optional registered as laptop hardware
- [ ] Door bell  - Mesecons or a bell sound on press
- [ ] Freezer - like a chest
- [ ] Microwave / Oven - like furnance
- [ ] Oven overhead - working sound and light
- [ ] Smoke detector - Search for fire nodes and start alarm sound if found
- [ ] Stereo - Play random music sound?
- [ ] Toaster - do toasts (support for waffles mod)
- [ ] Toilet - flush sound on stand up

**Comment by bell07** [comment]
THe "like a chest" should be easy to do if https://github.com/minetest/minetest_game/pull/2127 is merged

**Comment by bell07** [comment]
I seen you added some chestlike-code to the mod. The first test using ma_pops_furniture:end_table_aspen_wood filled my log with message:
`ERROR[Main]: GUIFormSpecMenu::drawMenu() Draw backgrounds unable to load texture:`

looked to the code, the reason is the `'background[4,4;0,0;]'..`
Eather you add a background image to this statement (`'background[4,4;0,0;mp_any_background_image.png]'.. ` or you just remove this line

**Comment by Gerold55** [comment]
Fixed it

**Comment by Gerold55** [comment]
For the stereo idk where I can get the music for it.

**Comment by bell07** [comment]
Same for ma_pops_furniture:counter

**Comment by Gerold55** [comment]
I've done a lot of work and added some more functionality to the furniture mod.

**Comment by Gerold55** [comment]
Is it possible to help me with the microwave and the oven when you get a chance to do so? I have an idea of how they should work so... @bell07

**Comment by bell07** [comment]
Let me know your idea



Gesendet von myMail für iOS


Dienstag, 29. Mai 2018, 17:33 +0200 von notifications@github.com  <notifications@github.com>:

**Comment by bell07** [comment]
Items should be restricted to food. I do not like without fuel or other restrictions. But maybe mesecons-enabled and with timer? The oven should not take the furnace obsolete

**Comment by Gerold55** [comment]
Yeah it should be for food only. I'm making the microwave without fuel. The
oven doesn't need to be without fuel though.

**Comment by bell07** [comment]
I looked to your last changes and finally lost interest for this mod. Now it is furniture + foods mod? What is coming next? I do not like to support such all-in-one mods. I like a food mod for food, a sofa mod for sofa, a freezer mod for freezer and so on.
So sorry, I cannot help you

**Comment by Gerold55** [comment]
I may get rid of the foods. I just implemented the freezer mod into the furniture mod will change it later.

**Comment by Gerold55** [comment]
What I'm doing is implementing working furniture and not just plain old storage.

**Comment by Gerold55** [comment]
@bell07 I've lightened up on the "food" part it was added originally from the original freezer mod so it is there but when certain mods are enabled it will work to freeze them (freezer) or cook them (microwave)

**Comment by Gerold55** [comment]
@bell07 I was wondering if you could help with a node timer and button?

**Comment by Gerold55** [comment]
[microwave2.zip](https://github.com/Gerold55/ma_pops_furniture/files/2117574/microwave2.zip)
That's the file that I'm talking about that I need the button and node timer for.

**Comment by bell07** [comment]
Sorry, at the moment my focus is not on minetest modding
